### DISCUSSION TITLE
How disable OVOS skill that adjusts the volume

### DISCUSSION THREAD

**Discussion by geetee24** [OpenVoiceOS/ovos-buildroot#270] [Q&A] [answered]
I want to disable: "set volume to maximum".  Where is that code/skill located?

**Comment by JarbasAl** [comment]
https://github.com/OpenVoiceOS/skill-ovos-volume

**Comment by geetee24** [comment]
how do i unload this skill?

**Comment by JarbasAl** [comment]
uninstall it, or blacklist it in mycroft.conf

OVOS does not force skills and doesnt install them again on boot by itself like mycroft did. 

dont want a skill? dont install it

**Comment by geetee24** [comment]
tell me how to uninstall it from the command line or blacklist it in the my...conf (what is the path)

**Comment by JarbasAl** [comment, marked as answer]
you can uninstall it by `pip uninstall ovos-skill-volume`, assuming you are not using docker

you can blacklist it by adding the skill_id to your config https://github.com/OpenVoiceOS/ovos-config/blob/dev/ovos_config/mycroft.conf#L214, the skill_id is `skill-ovos-volume.openvoiceos`

the config is at `~/.config/mycroft/mycroft.conf`, if it doesnt exist, create the file and add only the values you want to change  (the default values come from link above). config docs [here](https://openvoiceos.github.io/ovos-technical-manual/config/) for reference
